buildenv/jenkins/common/variables-functions

```diff
@@ -360,20 +360,6 @@ def set_node(job_type) {
     }
 }
 
-/*
-* Set the RELEASE variable with the value provided in the variables file.
-*/
-def set_release() {
```

Comment:
this method is still required - see stage('Java Version'){...} in common/build script